src/Controls/src/Core/SeparatorMenuItem.cs

```diff
@@ -1,6 +0,0 @@
-namespace Microsoft.Maui.Controls
-{
-	internal class SeparatorMenuItem : BaseMenuItem
```

Comment:
This type was internal and not used anywhere. Probably a legacy internal type from Xamarin.Forms that never got deleted.